codegenerator/cli/npm/envio/src/Internal.res

```diff
@@ -72,3 +81,44 @@ type eventItem = {
   //This gets set to true and does not try and reload events
   hasRegisteredDynamicContracts?: bool,
 }
+
+type fuelEventKind =
```

Comment:
Probably makes sense to namespace these into a "Fuel" module but not a big deal either way.